jekyll/_cci2/persist-data.md

```diff
@@ -39,74 +28,92 @@ For more information see the [Caching Dependencies]({{site.baseurl}}/2.0/caching
+
+Details about your storage and network transfer usage can now be viewed on your Plan Settings.
+
+### How to manage your storage and network transfer use
+
+There are several common ways that your configuration can be optimized to ensure you are getting the most out of your storage and network usage. 
+
+Before attempting to reduce data usage, you should first consider whether that usage is providing enough value to be kept. In the cases of caches and workspaces this can be quite easy to compare - does the developer/compute time saving from the cache outweigh the cost of the download and upload? Please see below for examples of storage and network optimization opportunities.
+
+### Opportunities to reduce artifact and cache/workspace traffic
+
+#### Check which artifacts are being uploaded
+
+Often we see that the store_artifacts step is being used on a large directory when only a few files are really needed, so a simple action you can take is to check which artifacts are being uploaded and why.
+
+If you are using parallelism in your jobs, it could be that each parallel task is uploading an identical artifact. You can use the CIRCLE_NODE_INDEX environment variable in a run step to change the behaviour of scripts depending on the parallel task run.
+
+#### Caching unused or superfluous dependencies
+
+Depending on what language and package management system you are using, you may be able to leverage tools that clear or “prune” unnecessary dependencies. For example, the node-prune package removes unnecessary files (markdown, typescript files, etc.) from node_modules.
+
+#### Optimizing cache usage
+
+If you notice your cache usage is high and would like to reduce it, try:
 
-**Uploading the same artifact many times**
+* Ensuring that your cache “key” is following best practices:
 
-Ensure that you are making intentional use of the `store_artifacts` step. Also
-check that parallelized jobs aren't uploading artifacts unless necessary.
+```sh
+     - save_cache:
+         key: brew-{{epoch}}
+         paths:
+           - /Users/distiller/Library/Caches/Homebrew
+           - /usr/local/Homebrew
+```
 
-**Caching unused or superfluous dependencies**
+Notice in the above example that best practices are not being followed. `brew-{{ epoch }}` will change every build; causing an upload every time even if the value has not changed. This will eventually cost you money, and never save you any time. Instead pick a cache key like the following:
 
-Depending on what language and package management system you are using, you may
-be able to leverage tools that clear or "prune" unnecessary dependencies. For
-example, the `node-prune` [package](https://github.com/tj/node-prune) removes
-unnecessary files (markdown, typescript files, etc.) from `node_modules`.
+```sh
+     - save_cache:
+         key: brew-{{checksum “Brewfile”}}
+         paths:
+           - /Users/distiller/Library/Caches/Homebrew
+           - /usr/local/Homebrew
+```
```

Comment:
```suggestion
{% raw %}
```sh
     - save_cache:
         key: brew-{{checksum “Brewfile”}}
         paths:
           - /Users/distiller/Library/Caches/Homebrew
           - /usr/local/Homebrew
```
{% endraw %}
```

Reply:
You'll need raw tags because of the bracket interpolation on  line 102. Should fix the red build.